app/src/main/java/com/hover/stax/ApplicationInstance.java

```diff
@@ -23,5 +24,7 @@ public void onCreate() {
 		Lingver.init(this, Locale.getDefault());
 
 		FirebaseApp.initializeApp(this);
+		FirebaseMessaging.getInstance().setAutoInitEnabled(true);
+		FirebaseMessaging.getInstance().setDeliveryMetricsExportToBigQuery(true);
```

Comment:
Do we need this?

Reply:
Export metrics to big query is also available on Firebase on the fly might be useful especially when our user base grow exponential.

Reply:
Don't enable things until they are needed. We already have to many data and analytics tools going on